authoring-new-actions.md

```diff
@@ -0,0 +1,90 @@
+# GitHub Actions
+
+[GitHub Actions](https://help.github.com/en/articles/about-github-actions) is a community-led and community-powered approach to enable developers to automate their code to cloud workflows directly from their repositories.
+There are now over 6,700+ community-developed actions available in [GitHub Marketplace](https://github.com/marketplace). 
+Since releasing Actions in November 2017, GitHub partners including Cloud partners like AWS, Azure, Google, and many others have also contributed to this list to help developers(teams) can extend and automate workflows with their existing tools.
```

Comment:
nit: help developers(teams) extend and automate

Reply:
corrected